msos/NativeStructs.cs

```diff
@@ -0,0 +1,926 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using DWORD = System.Int32;
+
+namespace msos
+{
+    static class NativeStructs
```

Comment:
A lot of stuff in this class also seems unnecessary now that we put everything back in `DumpReader`.